RestaurantReservation.API/Validators/CustomValidators.cs

```diff
@@ -0,0 +1,104 @@
+﻿using FluentValidation;
+using RestaurantReservation.API.Utilities;
+using RestaurantReservation.API.ValidationMessages;
+using System.Text.RegularExpressions;
+
+namespace RestaurantReservation.Api.Validators;
+
+/// <summary>
+/// Set of Extension methods for IRuleBuilder
+/// </summary>
+public static class CustomValidators
+{
+    public static IRuleBuilderOptions<T, string> ValidName<T>(this IRuleBuilder<T, string> ruleBuilder)
```

Comment:
i believe this class is too generic and doesn't adhere to the SRP, as it contains validation rules for multiple fields. plz consider creating separate validators for each field, so each field has its own dedicated validator like `PhoneNumberValidator` 
`PasswordValidator`